src/CMakeLists.txt

```diff
@@ -1,8 +1,19 @@
 find_package(PkgConfig REQUIRED)
-pkg_check_modules(EventdEvent REQUIRED libeventd-event>=0.3)
+pkg_check_modules(EventdEvent REQUIRED libeventd>=0.3)
 pkg_check_modules(EventdPlugin REQUIRED libeventd-plugin>=0.3)
 pkg_check_modules(SystemdJournal REQUIRED libsystemd-journal)
 
+macro(pkg_config_variable PKG VAR OUT)
+    execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE} --variable=${VAR} ${PKG}
+        OUTPUT_VARIABLE ${OUT})
+    string(STRIP ${${OUT}} ${OUT})
```

Comment:
I can understand if you've been exposed to old CMake code, but I find it *way* more usable than autoconf :) . There's magic, but the magic in CMake has names for it :P .